### PR TITLE
Add format-text file

### LINKED ISSUE #883: [FEAT] - Add re-usable "formatted text table" that can be used to format "tables" in chat
### What feature would you like to see?

Right now there are some commands, such as this one:
image

that would look better in a formatted table, using just "text" characters like `|` and `-` as borders of the table's rows and columns.

This utility should be re-usable and just return a large string, that can be passed into an embed and displayed more or less as-is. We could also just use a library to perform this for us as well. 

### Additional information

There are examples out there, but I can't find any right now.

### PR DESCRIPTION
**Pull request by bradtaniguchi**
# Pull Request

## Description

- Add format-text utility function

## Related Issue

Closes #883

## Documentation

For _any_ version updates, please verify if the [documentation page](https://www.beccalyria.com/discord-documentation) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [ ] My contribution does NOT require a documentation update.
- [X] My contribution DOES require a documentation update.

---

I wanted to get this out as a draft to get feedback, and to also get some help. As I have some tests, I'm unsure how to go about setting them up. I also wanted to possibly refactor the code to be more readable, but I don't want to do it without the tests to prevent any breakage.

### CODE REVIEW

**Comment by nhcarrigan** [comment]
How did you write the tests?

We have Mocha/Chai already in place - could add a new `.spec.ts` file to `tests` and run tests w/ Chai's `assert`.

**Comment by bradtaniguchi** [comment]
> How did you write the tests?
> 
> We have Mocha/Chai already in place - could add a new `formatter.spec.ts` file to `tests` and run tests w/ Chai's `assert`.

I wrote the tests in `jest`, but it shouldn't be too bad to migrate them over.
Here they are within replit (where I also lost some of the tests :'[ )

https://replit.com/@bradtaniguchi/format-table-text?v=1

**Comment by bradtaniguchi** [comment]
Ok cool, I'll add them and move em over tomorrow. Pretty burnt out from earlier today.

**Comment by nhcarrigan** [comment]
No rush! Let me know if you need any help.

File: test/formatText.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        }
+      )
+    ).to.eq(
+      "name | aggeeeeeeeee\n" +
+        "-------------------\n" +
+        "brad | 100         \n" +
+        "foo  | bar         "
+    )
+  );
+});
```
[the remark this answers is not in the document]
Comment (bradtaniguchi) [reply test/formatText.spec.ts:53]: I haven't forgotten about these, even though I missed the end of hacktoberfest.

I'll spend some time later today to finish migrating these over.

**Comment by bradtaniguchi** [comment]
Bah I just realized the changes requested where the ones you gave already. Just had to hit the button :/

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Pull request merged**